### PR TITLE
[SRP] Expose restorepolicy.minRestoreTime and depercate restorepolicy…

### PR DESCRIPTION
**Pull request by xmbms**
….lastEnabledTime

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json [hunk 1]
```diff
@@ -1260,7 +1260,13 @@
           "type": "string",
           "format": "date-time",
           "readOnly": true,
-          "description": "Returns the date and time the restore policy was last enabled."
+          "description": "Deprecated in favor of minRestoreTime property."
```
Comment (blueww) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json:1263]: Please Update Example

**Comment by qianwens** [comment]
@xmbms could you please help update the example for the change and let me know when it is ready to merge.

**Review by blueww — APPROVED** [review]

**Comment by blueww** [comment]
I am OK with it, as .net SDK pass.
Anyway, this might be breaking change as new property added. 
If ARM review board approve it, I am OK for merge it.

**Pull request merged**